blueprints/modifier-test/index.js

```diff
@@ -38,16 +38,4 @@ module.exports = useTestFrameworkDetector({
       dasherizedModulePrefix: dasherizedModulePrefix,
     };
   },
-
-  afterInstall: function (options) {
-    if (
-      !options.dryRun &&
-      options.testType === 'integration' &&
-      isPackageMissing(this, 'ember-cli-htmlbars-inline-precompile')
-    ) {
-      return this.addPackagesToProject([
-        { name: 'ember-cli-htmlbars-inline-precompile', target: '^0.3.1' },
-      ]);
-    }
-  },
```

Comment:
No longer needed, since we can reasonably assume that the host app/addon has `ember-cli-htmlbars` itself installed already.